pol-core/support/scripts/ecompile.cfg.example

```diff
@@ -5,25 +5,26 @@
 #
 # ModuleDirectory: Location of *.em files
 #
-ModuleDirectory=D:\DistroDev\scripts\modules
+ModuleDirectory=/scripts/modules
```

Comment:
Aren't these technically still "absolute"?

Reply:
If no absolute path is specified POL handles them as relative to POL root, or at least it used to. I haven't tested with Cores later than 098.